src/components/ui/AvatarInput/AvatarInput.tsx

```diff
@@ -60,11 +61,14 @@ const AvatarInput = ({
       </label>
       <label
         htmlFor="userAvatar"
-        className={buttonVariant({
-          size: 'md',
-          variant: 'secondary',
-          className: buttonStyles,
-        })}
+        className={clsx(
+          styles.uploadButton,
+          buttonVariant({
+            size: 'md',
+            variant: 'secondary',
+            className: buttonStyles,
+          }),
+        )}
```

Comment:
You can concat styles in className buttonVariant props directly, but also i think you need to make it more straightforward